eurosurveillance.csl

```diff
@@ -0,0 +1,265 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en">
```

Comment:
I think it requires a specific English language.
en-GB or en-US are the most common ones.
Note that they can change your style, especially how dates are shown.

Reply:
I do not understand your comment, as I had later on a message "All checks have passed" without changing the default-locale